routes/api_v1_0.py

```diff
@@ -469,10 +470,14 @@ def get_data_v1_0():
 
         bearings = None
         if "mag" in result["variable"]:
+            bearings_var = DatasetConfig(
+                result["dataset"]).variable[result["variable"]].bearing_component
```

Comment:
You can use the `DatasetConfig` object that's already been instantiated on line 447!
```suggestion
            bearings_var = config.variable[result["variable"]].bearing_component
```